grafana.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Grafana",
+    "plugin_schema": "2",
+    "release": "11.3-RELEASE",
+    "artifact": "https://github.com/MalteHillmann/iocage-plugin-grafana.git",
+    "pkgs": [
+        "grafana6",
+        "influxdb",
+        "curl"
+    ],
+    "properties": {
+        "dhcp": 1
+    },
+    "packagesite": "http://pkg.FreeBSD.org/FreeBSD:11:amd64/latest",
```

Comment:
Can you use "http://pkg.FreeBSD.org/${ABI}/latest" instead so that it works on upcoming 12.0 as well?